pkgs/by-name/wp/wpaperd/package.nix

```diff
@@ -2,16 +2,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "wpaperd";
-  version = "1.0.1";
+  version = "1.1.0";
```

Comment:
Version `1.1.1` was just released. That solves https://github.com/danyspin97/wpaperd/issues/104